chat-client/src/client/chat.test.ts

```diff
@@ -39,8 +44,23 @@ describe('Chat', () => {
         })
     })
 
-    it('publishes ready event, when initialized', () => {
-        assert.calledOnceWithExactly(clientApi.postMessage, { command: READY_NOTIFICATION_METHOD })
+    it('publishes ready event and initial tab add event, when initialized', () => {
+        sinon.assert.callCount(clientApi.postMessage, 3)
+
+        sinon.assert.calledWithExactly(clientApi.postMessage.getCall(0), { command: READY_NOTIFICATION_METHOD })
```

Comment:
nit: I think the following syntax is also possible and I personally prefer it on tests:

`clientApi.postMessage.firstCall`
`clientApi.postMessage.secondCall`
`clientApi.postMessage.thirdCall`


So the assertions would look like:   `sinon.assert.calledWithExactly(clientApi.postMessage.firstCall, { command: READY_NOTIFICATION_METHOD })`

Reply:
Yeah, that's more readable